.sass-lint.yml

```diff
@@ -11,7 +11,7 @@ rules:
   no-url-protocols: 0
   class-name-format:
     - 1
-    - convention: ^([a-z0-9]+([A-Z][a-z0-9]+)*)(-{1}([a-z0-9]+([A-Z][a-z0-9]+)*)){0,2}$
+    - convention: ^([a-z0-9]+([A-Z][a-z0-9]+)*)(-{1}([a-z0-9]+([A-Z][a-z0-9]+)*)){0,4}$
     - convention-explanation: 'Classes should be camelCase and separated by no more than 5 hyphens'
```

Comment:
@amberRrucker Sorry I referenced the wrong thing, but what we're suggesting is something like this: 

Instead of doing this:
```
.fa-google-plus-square {
   &::before {
     color: $googleplus-swatch;
   }
 }
```

Do this:

```
.faBefore-googleplus {
   &::before {
     color: $googleplus-swatch;
   }
 }
```

And then add the class to `<i class="fa fa-2x fa-google-plus-square fa-stack-2x"></i>`

And for consistency, just do that for all the classes.